deny.toml

```diff
@@ -1,27 +1,41 @@
 [licenses]
 allow = [
-  "MIT",
+  "0BSD",
+  "Apache-2.0 WITH LLVM-exception",
+  "Apache-2.0",
+  "BSD-2-Clause",
+  "BSD-3-Clause",
+  "BSL-1.0",
   "CC0-1.0",
   "ISC",
+  "MIT",
   "OpenSSL",
   "Unicode-3.0",
+  "Unicode-DFS-2016",
   "Unlicense",
-  "BSD-2-Clause",
-  "BSD-3-Clause",
-  "Apache-2.0",
-  "Apache-2.0 WITH LLVM-exception",
   "Zlib",
 ]
 
-unlicensed = "warn"
```

Comment:
These config options default to `error` now after https://github.com/EmbarkStudios/cargo-deny/pull/611